source/enable-autocomplete.txt

```diff
@@ -0,0 +1,98 @@
+.. _vs-code-enable-autocomplete:
+
+====================
+Enable Auto Complete
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can enable autocomplete for MongoDB Playgrounds. Enabling 
+autocomplete allows you to automatically populate database and collection 
+names when using ``use("")`` or 
+``db.getCollection("")`` syntax within a MongoDB playground.
+
+Before you Begin
+----------------
+
+Install the :ref:`MongoDB VSCode extension <vsce-install>`.
+
+Steps
+-----
+
+.. procedure::
+   :style: connected
+
+   .. step:: Open Settings.json
+     
+      In VSCode, open the command palette:
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 30 70
+
+         * - Operating System
+           - Keyboard Command
+
+         * - Windows
+           - :kbd:`CTRL` + :kbd:`SHIFT` + :kbd:`P`
+
+         * - MacOS
+           - :kbd:`CMD` + :kbd:`SHIFT` + :kbd:`P`
+
+      Enter :guilabel:`Open User Settings (JSON)` and select the option 
+      from the autocompleted list:
+
+      .. figure:: /images/vsce-command-palette-json-settings.png
+         :figwidth: 700px
+         :alt: Image of a stream connection
+
+   .. step:: Add the autocomplete snippet to the ``Settings.json`` file
+     
+      Copy the following code snippet:
+
+      .. code-block::
+
+         "editor.quickSuggestions": {
+            "other": true,
+            "comments": false,
+            "strings": true
+         }
+
+      Paste the code into the ``Settings.json`` file:
+
+      .. figure:: /images/vsce-add-snippet-json-settings.png
+         :figwidth: 700px
+         :alt: Image of a stream connection
+
+   .. step:: Test the autocompletion feature
+
+      In a new playground file connected to your deployment, 
+      type the first two characters of a collection name between  
```

Comment:
This functionality works for me after typing a single letter for a collection name.

<img width="577" alt="Screenshot 2024-04-08 at 11 34 41 AM" src="https://github.com/mongodb/docs-mongodb-vscode/assets/29379027/cb1978f1-0ce6-4868-a096-5560023cba92">

Reply:
Updated the first example verbiage and screenshot to show that only a single character is necessary here.